crates/utils/src/request.rs

```diff
@@ -47,31 +48,61 @@ where
   response.expect("retry http request")
 }
 
-#[derive(Deserialize, Debug)]
-pub struct IframelyResponse {
+#[derive(Deserialize, Serialize, Debug, PartialEq)]
+pub struct PostLinkTags {
```

Comment:
Is the term `PostLink` or `PostLinkTags` a common term, or something you came up with? To me it sounds very awkward, just using `OpenGraph` would probably be better.

Reply:
I don't want to use OpenGraph alone, because we use regular  html tags, and there's potential we'd use other tags too. I use `HtmlTags` on the front end, would that be better?

Reply:
Better, or how about `SiteMetadata` or similar?

Reply:
That works.